cmd/monitor.go

```diff
@@ -53,45 +55,172 @@ func NewMonitorItemsListCmd() *cobra.Command {
 	return cmd
 }
 
-// Upload files to Monitor
+var monitorItemsDownloadCmdHelp = `Download files from your account.
 
-type monitorFileUpload struct {
-	uploader *vt.MonitorUploader
-}
+This command download files in your monitor account using their MonitorItemID.`
+
+var monitorItemsDownloadCmdExample = `  vt monitor items download "MonitorItemID"
```

Comment:
For coherency with the `download` command, which is `vt download` instead of `vt files download` I would make this command `vt monitor download`. In fact, all the the commands hanging from `items` could hang from `monitor` and that users would save keystrokes. Unless you are planning to include some other monitor command that is not related to "items", I would remove the `items` command.